Update General Elment and URL

piquark6046 opened the pull request:
https://github.com/AdguardTeam/AdguardFilters/issues/88215

Commit:
Update General URL

Commit:
bosa.co.kr

Commit:
Merge branch 'master' into bosacokr-20210713-1

Commit:
Fix missed

Commit:
Update General Elemhide

Commit:
Fix missed

piquark6046 commented on EnglishFilter/sections/general_url.txt:
```diff
@@ -81,6 +81,7 @@ porn*.net/bees.js|
 !+ NOT_OPTIMIZED
 /getads?
 !
+/bannerpop/uploads/*$image,script
```
This rule can apply to many website.
https://www.todaykorea.co.kr/
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://datanet.co.kr/
https://www.entermedia.co.kr/ ( https://www.entermedia.co.kr/news/articleView.html?idxno=27002 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
http://www.constimes.co.kr/
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )

piquark6046 commented on EnglishFilter/sections/general_elemhide.txt:
```diff
@@ -97,6 +97,7 @@
 ##.div-gpt-ad:not([style^="width: 1px; height: 1px;"])
 ##[class^="div-gpt-ad"]:not([style^="width: 1px; height: 1px;"])
 !
+##div[class^="banner_"]
```
This rule also can apply to many website.
https://www.bosa.co.kr/news/articleView.html?idxno=2154214
https://www.econovill.com/ ( https://www.econovill.com/news/articleView.html?idxno=540458 )
https://www.sisafocus.co.kr/news/articleView.html?idxno=262932
http://www.newsian.co.kr/ ( http://www.newsian.co.kr/news/articleView.html?idxno=49785 )
https://www.idsn.co.kr/ ( https://www.idsn.co.kr/news/articleView.html?idxno=50752 )
https://www.ggilbo.com ( https://www.ggilbo.com/news/articleView.html?idxno=858021 )
http://www.andongdaily.com/ ( http://www.andongdaily.com/news/articleView.html?idxno=24475 )
https://www.tiptipnews.co.kr/news/articleView.html?idxno=80869
http://www.fntoday.co.kr/ ( http://www.fntoday.co.kr/news/articleView.html?idxno=260316 )

Alex-302 replied:
This rule is extremely dangerous) Please make domain specific.

Alex-302 commented on EnglishFilter/sections/general_url.txt:
```diff
@@ -81,6 +81,7 @@ porn*.net/bees.js|
 !+ NOT_OPTIMIZED
 /getads?
 !
+/bannerpop/uploads/*$image,script
```
It seems modifiers are unnecessary in this case. Just `/bannerpop/uploads/*`

piquark6046 replied:
https://github.com/AdguardTeam/AdguardFilters/pull/88218/commits/2023e9a687a9d5b40c3d189ee58ecddc5ff55488

Commit:
Moved to specific.txt

Pentanium/ABClientFilters@0225ab3c4fe848c2fdcfdd957fe834d24abf5a22
Pentanium/ABClientFilters@16f405b71d5510618a6d2ac9c31915654f0cf3a7
Pentanium/ABClientFilters@c711e31c6a520b33853cb31167967734cc397836
Pentanium/ABClientFilters@f209ca3563cb276d52592fdfd801c9ce8e804b6d
Pentanium/ABClientFilters@3c2d4e07730c0eea8df0784f73b10b63afe8c727
Pentanium/ABClientFilters@46b8da24024b00c29cee9f973aef887b0e284b83
Pentanium/ABClientFilters@83bd49229a3ff26f431c1d4c29745f071d2929a1
Pentanium/ABClientFilters@87d879d2f4b4c63c9db0dbdb59a7bb409cca1a31
Pentanium/ABClientFilters@6758e268608c4c0af9ca8d4e8472275874a8ba94
Pentanium/ABClientFilters@cada4b8a704554095fa6f64e1afad7dfbfb85d9f
Pentanium/ABClientFilters@fbf7ab339bc2c4da4ad7bd06b58cfc818d62c346
Pentanium/ABClientFilters@bc67959312988183842244178c040f6542b5db52
Pentanium/ABClientFilters@b4ecefc3a1531102e59ada1c42ce0e3c855ac23e
Pentanium/ABClientFilters@0f852d89e5a1da6c8c8bf6dbe400a1228707c06f
Pentanium/ABClientFilters@cdeef8b1bf34310b95b338e209fd325a8864c756
Pentanium/ABClientFilters@659c44e7bfe63ef3ea77155a5420b5f78a05e2e4
Pentanium/ABClientFilters@73c8dcd3250b19871be6bffe382de86a818e66ea
Pentanium/ABClientFilters@bc67959312988183842244178c040f6542b5db52

piquark6046 commented:
Oh, I forgot leaving `Co-authored-by` in https://github.com/AdguardTeam/AdguardFilters/pull/88218/commits/2023e9a687a9d5b40c3d189ee58ecddc5ff55488 commit.

piquark6046 commented:
@Alex-302 What about this rule instead of https://github.com/AdguardTeam/AdguardFilters/pull/88218/files#diff-fbf56167647d1d9892a9a9ccdd71a6c88bdf70c0d33822f99e0270e936c4b898R4350-R4351 ?
`##div.banner_box`
